Slider/Assets/Scripts/Player/PlayerAction.cs

```diff
@@ -173,6 +173,44 @@ private bool AttemptItemPickup()
         return false;
     }
 
+    private Item GetNearestValidItem(List<Collider2D> collidersInRange)
+    {
+        collidersInRange.OrderBy((item) => Vector3.Distance(item.transform.position, transform.position))
+                                         .Select((collider) => collider.GetComponent<Item>());
+        foreach(Collider2D c in collidersInRange) {
+            if(CheckItemRaycast(c))
+                return c.GetComponent<Item>();
+        }
+        
+        return null;
+    }
+
+    private bool CheckItemRaycast(Collider2D item)
+    {
+        Vector3 itemPos = item.transform.position;
+        Vector3 direction = itemPos - transform.position;
+        Vector3 perp = Vector3.Cross(direction, Vector3.forward).normalized * 1/2;
```

Comment:
can you make the 1/2 a parameter in case we want to change the width of items. specifically we should probably use the items collider radius minus some small amount (0.1).

ex. the village bobs are radius 0.4 instead of 0.5 bc of the sprite size